drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -95,6 +95,13 @@ def resolve(input:)
         if record.is_a? Hmis::Hud::HmisService
           record.owner.save! # Save the actual service record
           record = Hmis::Hud::HmisService.find_by(owner: record.owner) # Refresh from View
+        elsif record.is_a? HmisExternalApis::AcHmis::ReferralRequest
+          begin
+            HmisExternalApis::AcHmis::CreateReferralRequestJob.perform_now(record)
+          rescue HmisErrors::ApiError => e
+            errors.add :base, :server_error, full_message: e.message
+            return { errors: errors }
+          end
```

Comment:
I don't think it's necessary to rescue and return these, we can just allow it to raise.